docs/concepts/privacy.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Privacy
+sidebarDepth: 0
+description: Learn about user privacy in IPFS and why it does not come with a built-in privacy layer or encryption.
+related:
+  'Article: IPFS Privacy (Textile)': https://medium.com/pinata/ipfs-privacy-711f4b72b2ea
+---
+
+# IPFS and privacy
+
+As a protocol for peer-to-peer data storage and delivery, IPFS is by design a _public network_: Nodes participating in the network store data affiliated with globally consistent [content addresses](/concepts/content-addressing) (CIDs) and advertise that they have those CIDs available for other nodes to use through publicly viewable [distributed hash tables](/concepts/dht/) (DHTs). This paradigm is one of IPFS's core strengths — at its most basic, it's essentially a globally distributed "server" of the network's total available data, referenceable both by the content itself (those CIDs) and by the participants (the nodes) who have or want the content.
+
+What this does mean, however, is that IPFS itself isn't explicitly protecting knowledge _about_ CIDs and the nodes that provide or retrieve them. This isn't something unique to the distributed web; on both the d-web and the legacy web, traffic and other metadata can be monitored in ways that can infer a lot about a network and its users. Some key details on this are outlined below, but in short: While IPFS traffic _between nodes_ is encrypted, the metadata those nodes publish to the DHT is public. Nodes announce a variety of information essential to the DHT's function — including their unique node identifiers (PeerIDs) and the CIDs of data that they're providing — and because of this, information about which nodes are retrieving and/or reproviding which CIDs is publicly available.
```

Comment:
I believe @johnnymatthews did this - Johnny, is this house style now?

Reply:
Yeah, I thought I'd sneak this in. I'm really not a fan of `dweb` because it reads like _dweeb_. But every time the whole `dweb`, `d-web`, `Dweb` discussion comes up, we never really seem to come to an answer. So I've just been covertly using `d-web`.

So I guess in a sense, `d-web` is the house style. But not by popular vote, or any vote for that matter.

Reply:
For the record, if this was intentional, I don't feel either way about this, feel free to ignore this and keep as `d-web` :)

Reply:
The cynic in me says "LET'S TAKE BACK 'DWEEB'!!1!" but totally happy to defer on this one ;)